src/Mount.cpp

```diff
@@ -664,6 +702,81 @@ bool Mount::connectToDriver( TMC2209Stepper* driver, const char *driverKind ) {
 #endif
 #endif
 
+/////////////////////////////////
+//
+// configureFocusdriver
+// TMC2209 UART only
+/////////////////////////////////
+#if (FOCUS_STEPPER_TYPE != STEPPER_TYPE_NONE) && (FOCUS_DRIVER_TYPE == DRIVER_TYPE_TMC2209_UART)
+#if SW_SERIAL_UART == 0
+  void Mount::configureFocusDriver(Stream *serial, float rsense, byte driveraddress, int rmscurrent, int stallvalue)
+  {
+    _driverFocus = new TMC2209Stepper(serial, rsense, driveraddress);
+    _driverFocus->begin();
+    bool _UART_Rx_connected = false;
+    #if UART_CONNECTION_TEST_TXRX == 1
```

Comment:
Can you refactor this part to something like:
```C++
    #if UART_CONNECTION_TEST_TXRX == 1
        const bool UART_Rx_connected = connectToDriver( _driverFocus, "Focus" );
        if (!UART_Rx_connected) {
            digitalWrite(FOCUS_EN_PIN, HIGH);    //Disable motor for safety reasons if UART connection fails to avoid operating at incorrect rms_current
        }
    if (UART_Rx_connected){
        LOGV2(DEBUG_STEPPERS, F("Mount: Actual Focus motor rms_current: %d mA"), _driverFocus->rms_current());
        LOGV2(DEBUG_STEPPERS, F("Mount: Actual Focus CS value: %d"), _driverFocus->cs_actual());
        LOGV2(DEBUG_STEPPERS, F("Mount: Actual Focus vsense: %d"), _driverFocus->vsense());
    }
    #endif
```

- Remove leading underscore for local variable (it's a reserved identifier)
- Move local var into `#if` block since that's the only place where it's used

Reply:
Can't do that since the driver initialization needs to run before you can output the 3 values.